### DISCUSSION TITLE
Parameterized variants

### DISCUSSION THREAD

**Discussion by idrm** [tailwindlabs/tailwindcss#9076] [Help] [answered]
I suspect the answer is no, but is it possible to use the plugin system to create parameterized variants? I'm specifically interested in implementing such a variant that targets arbitrary attributes. 

For example, if such a variant were named `attr`, then `attr-[data-disabled]:(text-gray-100)` would be transformed to `&[data-disabled]: { color: gray; }`.

**Comment by wongjn** [comment, marked as answer]
You could use [arbitrary variants](https://tailwindcss.com/docs/hover-focus-and-other-states#using-arbitrary-variants). In your case, your class would be `[&[data-disabled]]:text-gray-100`.

**Reply by idrm** [reply]
Excellent! Thanks for pointing this out.
